wa/commands/create.py

```diff
@@ -35,6 +44,171 @@
+        # Display the version if needed and exit
+        if args.schema_version:
+            self.logger.info(
+                'The current schema version is {}'.format(self.schemaversion))
+            return
+        # Create the database (override if exists)
+        if args.dbname == 'postgres':
+            raise ValueError('Databasename to create cannot be postgres.')
+        # Test passwordless connection
+        if not args.password:
+            try:
+                test_conn = connect(dbname='postgres', user=args.username,
+                                    host=args.postgres_host, port=args.postgres_port)
+                test_conn.close()
+            except OperationalError:
+                raise CommandError(
+                    'Passwordless connection to server failed.' +
+                    ' Change peer to trust in pg_hba.conf.' +
+                    ' Alternatively, supply a password as an argument or in .pgpass.')
+        # Test if database already exists
+        if not args.force:
+            try:
+                if args.password:
+                    test_conn = connect(dbname=args.dbname, user=args.username,
+                                        password=args.password, host=args.postgres_host, port=args.postgres_port)
+                else:
+                    test_conn = connect(dbname=args.dbname, user=args.username,
+                                        host=args.postgres_host, port=args.postgres_port)
+                test_conn.close()
+            except OperationalError:
+                raise CommandError(
+                    "Database {} already exists.".format(args.dbname) +
+                    "Please specify the -f flag to create it from afresh."
+                )
+
+        self.create_database(args)
+
+        # Update config file
+        with open(args.config_file, 'r') as config_file:
+            config = yaml.load(config_file)
+        if 'postgres' in config:
+            if not args.force_update_config:
+                self.logger.warning("The entry postgres already exists in the config file. " +
+                                    "Please specify the -F flag to force an update.")
+            if isinstance(config['postgres'], OrderedDict):
+                args_dict = vars(args)
+                for item in ['postgres_host', 'postgres_port', 'dbname', 'username', 'password']:
+                    if item in config['postgres']:
+                        if args.force_update_config:
+                            config['postgres'][item] = args_dict[item]
+                    else:
+                        config['postgres'][item] = args_dict[item]
+            else:
+                if args.force_update_config:
+                    config['postgres'] = OrderedDict([('postgres_host', args.postgres_host), ('postgres_port', args.postgres_port), (
+                    'dbname', args.dbname), ('username', args.username), ('password', args.password)])
```

Comment:
PEP8 (E122 continuation line missing indentation or outdented) fix for this line needed.